zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1564,7 +1564,63 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /forms:
+    get:
+      tags:
+        - Forms
+      summary: Return form by a Form Key
+      description: |
+        Get the form by the form key.
+
+        :::note
+        This endpoint will return only linked foms. Embedded forms are not supported.
```

Comment:
Typo `forms`. We need also a period after the sentence. For example:

`summary: Return form by a Form Key.`
Could you check other descriptions for the `.` ?